utils/utils.go

```diff
@@ -140,3 +143,16 @@ func InstallSshdConf() error {
 	fmt.Println("Restart sshd: systemd restart sshd")
 	return nil
 }
+
+func GetOwnerPassword() []byte {
+	for {
+		fmt.Printf("Enter owner password: ")
+		password, err := term.ReadPassword(int(syscall.Stdin))
```

Comment:
This would only allow us to enter the owner password through the terminal. Most likely you will be using this in the background so you need something like `pinentry`. It would be better to make this based off on how `pin` is currently fetched.